### DISCUSSION TITLE
Add option to search all sensors in PCAP interface

### DISCUSSION THREAD

**Discussion by UMHB-InfoSec** [Security-Onion-Solutions/securityonion#5343] [Ideas] [answered]
When adding a pcap job manually through the web ui could there be an option to search all sensor nodes in the cluster?

As it is I have 5 sensors and searching them independently is a slog.

Or ... am I doing something wrong : )

**Comment by dougburks** [comment, marked as answer]
Most folks use Hunt to find the stream that they're looking for and then pivot to PCAP from there. This allows you to easily search across your entire deployment.

For more information, please see https://docs.securityonion.net/en/2.3/hunt.html#context-menu.
